Woof/WoofTests/Model/KeyValueStorage/Extensions/KeyValueStorage+UnitTests.swift

```diff
@@ -4,6 +4,11 @@ extension KeyValueStorage {
         KeyValueStorage("Test-KeyValue-Storage")
     }
 
+    /// A storage for preferences
+    static var preferencesStorage: KeyValueStorage {
```

Comment:
I'd suggest to put the `test` prefix to avoid accidental incidents :)

```suggestion
    static var testPreferencesStorage: KeyValueStorage {
```

Reply:
Thanks for the helpful comment!
fixed